sdks/java/io/rabbitmq/src/test/java/org/apache/beam/sdk/io/rabbitmq/RabbitMqIOTest.java

```diff
@@ -185,20 +234,154 @@ public void testReadExchange() throws Exception {
       publisher.join();
     } finally {
       if (channel != null) {
-        channel.close();
+        // channel may have already been closed automatically due to protocol failure
```

Comment:
These both can be condensed into a single `IOUtils.closeQuietly` https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/IOUtils.html#closeQuietly(java.io.Closeable...)

Reply:
I looked at that but they actually cannot. Despite the convention of naming these `close` they do not actually implement `Closeable`.